repo/packages/P/prometheus/0/marathon.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "prometheus",
+  "cpus": 1,
+  "mem": 2048,
+  "ports": [9090],
```

Comment:
Yes, we can use a dynamic port. I just didn't understand marathon's API. It's fixed in the new PR (#10).